sei-devnet-2/genesis.json

```diff
@@ -0,0 +1,1475 @@
+            },
+            {
+                "date": "2023-04-21",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-04-28",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-05-05",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-05-12",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-05-19",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-05-26",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2024-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2025-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2026-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2027-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2028-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2029-06-01",
+                "token_release_amount": "1000000000"
+            }
+        ]
+      }
+    },
+    "nitro": {
+      "params": {
+        "whitelisted_tx_senders": []
+      },
+      "slot": "0",
+      "sender": "",
+      "stateRoot": "",
+      "txs": []
+    },
+    "oracle": {
+      "params": {
+        "vote_period": "10",
+        "vote_threshold": "0.500000000000000000",
+        "reward_band": "0.020000000000000000",
+        "whitelist": [
+          {
+            "name": "uatom"
+          },
+          {
+            "name": "uusdc"
+          },
+          {
+            "name": "usei"
+          },
+          {
+            "name": "ueth"
+          }
+        ],
+        "slash_fraction": "0.000100000000000000",
+        "slash_window": "201600",
+        "min_valid_per_window": "0.050000000000000000",
+        "lookback_duration": "3600"
+      },
+      "feeder_delegations": [],
+      "exchange_rates": [],
+      "penalty_counters": [],
+      "aggregate_exchange_rate_votes": [],
+      "price_snapshots": []
+    },
+    "params": null,
+    "slashing": {
+      "params": {
+        "signed_blocks_window": "10000",
+        "min_signed_per_window": "0.050000000000000000",
+        "downtime_jail_duration": "600s",
+        "slash_fraction_double_sign": "0.050000000000000000",
+        "slash_fraction_downtime": "0.010000000000000000"
+      },
+      "signing_infos": [],
+      "missed_blocks": []
+    },
+    "staking": {
+      "params": {
+        "unbonding_time": "1814400s",
```

Comment:
@Kbhat1 PTAL

Reply:
As discussed, lets change to 3 days, but lets add to checklist for atlantic-2 / mainnet on whether reducing to 3 days exposes us to risk of folks unbonding and running etc.